tests/installation/install_and_reboot.pm

```diff
@@ -22,32 +22,28 @@ sub run() {
     my @tags = qw/rebootnow/;
     if (get_var("UPGRADE")) {
         push(@tags, "ERROR-removing-package");
+        push(@tags, "DIALOG-packages-notifications");
         $timeout = 5500;    # upgrades are slower
     }
     while (1) {
         assert_screen \@tags, $timeout;
 
-        if (match_has_tag("popup-warning")) {
```

Comment:
I assume assert_screen @tags (tags = rebootnow; ERROR-removing-package) can never produce match_has_tag("popup-warning"), because one needle with popup-warning tag has no other tags:

~ grep -ri "popup-warning" needles-*
needles-sles/inst-internal-error-20151106.json:    "popup-warning"

~ grep -A2 tags needles-sles/inst-internal-error-20151106.json
  "tags": [
    "popup-warning"
  ]